elliott

```diff
@@ -335,10 +336,14 @@ advisory.
               metavar='ADVISORY_TYPE',
               type=click.Choice(['image', 'rpm', 'security']),
               help="Use the default value from ocp-build-data for ADVISORY_TYPE [image, rpm, security]")
-@click.option("--auto",
-              required=False,
-              default=False, is_flag=True,
-              help="Auto-find mode, adds bugs based on --group")
+@click.option("--mode",
+              required=True,
+              type=click.Choice(['LIST', 'SWEEP', 'PAYLOAD']),
+              help="""Mode to use to find bugs: 
+  LIST - Specify a list of BZ IDs to add
+  SWEEP - Find bugs based on --group
+  PAYLOAD - Find bugs fixed between two payloads. You must provide two payloads to compare --old and --new.
```

Comment:
`payload` is a kind of strange name for this. `mode` implies a verb to me. Sweeping is a verb, listing is a verb, but payload is a noun. Maybe call it `diff` ?